SCRIPTS.md

```diff
@@ -93,3 +93,4 @@
 | 89\.     | Voice-Assistant  | Voice-Assistant Bot | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Voice-Assistant) |  [Avinash .K. Rajan](https://github.com/avinashkranjan) |
```

Comment:
remove this modification. We are looking to automate this process and therefore this updation is not needed as of now

Reply:
@kaustubhgupta  as mentioned by you i have removed the modification please review and let me know if any further changes are required